src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/PublicationsAccordion.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import {
+  Accordion,
+  AccordionItem,
+  AccordionItemHeading,
+  AccordionItemPanel,
+  AccordionItemButton
+} from 'src/shared/components/accordion';
+import ExternalLink from 'src/shared/components/external-link/ExternalLink';
+
+import JSONValue from 'src/shared/types/JSON';
+import { EntityViewerSidebarPayload } from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarState';
+import { Publication } from 'src/content/app/entity-viewer/types/publication';
+
+import styles from './Overview.scss';
+
+type Props = {
+  sidebarPayload: EntityViewerSidebarPayload | null;
+  sidebarUIState: { [key: string]: JSONValue } | null;
+  updateEntityUI: (uIstate: { [key: string]: JSONValue }) => void;
+};
+
+const PublicationsAccordion = (props: Props) => {
+  const publications = props.sidebarPayload?.publications || [];
+  const expandedPanels = props.sidebarUIState?.publicationsAccordion
+    ?.expandedPanels as string[];
+
+  const onChange = (expandedPanels: (string | number)[] = []) => {
```

Comment:
why the default empty array?

Reply:
I added it to investigate some issue and forgot to remove it. Removed it now.